### DISCUSSION TITLE
Trying to use shadcn/ui in a WordPress theme

### DISCUSSION THREAD

**Discussion by moisreis** [shadcn-ui/ui#1373] [Q&A]
I'm a WordPress developer for now and I would like to integrate shadcn/ui in my theme. I am aware that WordPress supports React components using wp-scripts, but I just can't seem to do it. Did anybody try it?

**Comment by patrunks** [comment]
hello, it is possible to use shadcn ui in wordpress ?

**Reply by KakhnychOleh** [reply]
Hi! I'm also interested in this question, did you manage to implement this?

**Comment by rgbirkey** [comment]
I am also interested in how to integrate Shadcn with WordPress, and the DIVI theme if possible.
